Django_HW/users/admin.py

```diff
@@ -1,3 +1,9 @@
 from django.contrib import admin
+from .models import User
 
-# Register your models here.
+
+class UserAdmin(admin.ModelAdmin):
```

Comment:
nit: better to extend UserAdmin model 
`from django.contrib.auth.admin import UserAdmin`